my_list.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'my_enumerable'
+class Mylist
+  include MyEnumerable
+
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+new_list = Mylist.new([1, 2, 3, 4]) # true
+
+# Test #all?
+puts(new_list.all? { |e| e < 5 })
+puts(new_list.all? { |e| e > 5 })
+
+# Test #any?
+puts(new_list.any? { |e| e == 2 })
+puts(new_list.any? { |e| e == 5 })
+
+# Test #filter
+puts(new_list.filter(&:even?))
```

Comment:
Good job working on the project requirements. But when I tried running this command "ruby my_list.rb" I got an error. Please kindly look at the screen shot below:

 
![Screenshot 2023-10-14 at 6 55 58 AM](https://github.com/yusufyusie/Enumerable/assets/79506105/61be1e5e-3c8a-44d0-9c32-8e18da46d040)
It would be nice if you fixed this.